### PR TITLE
Topics/k1ch/admin GET: /personas/{persona_key}/roles

### PR DESCRIPTION
**Pull request by k1ch**
# Overview
- [feat: topics/k1ch/admin-get-personas-roles/introduce Admin](https://github.com/DMGT-TECH/the-usher-server/commit/b15e4fb1ba1481c6f678cba7d78b7441b9f663d4)
- [chore: topics/k1ch/admin-get-personas-roles/ API & DB tests](https://github.com/DMGT-TECH/the-usher-server/commit/752a8bf924ce98decd0bae83c1f1c45259c7b443)

<details>
  <summary>OAS Update: </summary>

image

</details>

<details open>
  <summary>New tests: </summary>

image

</details>

## Your PR Checklist 🚨

❤️ Please review the [guidelines for contributing](../docs/CONTRIBUTING.md) to this repository. Our goal is to merge PRs fast 💨 .

- [x] Check your code additions will fail neither code linting checks nor unit tests.
- [x] Additional units tests have been added to prove code updates work and fixes are effective
- [x] Additional documentation has been added (if appropriate)
- [x] Update Assignee field to yourself

## Issue Reference

Relates to #58

### CODE REVIEW

File: server/test/endpoint_admin_personas_roles.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+const { getAdmin1IdPToken, getTestUser1IdPToken } = require('./lib/tokens')
+const { getServerUrl } = require('./lib/urls')
+const { usherDb } = require('../../database/layer/knex')
+
+
+describe('Admin Personas Roles', () => {
+  let requestHeaders
+  let testPersonaKey
+  const url = `${getServerUrl()}`
```
Comment (mikelax) [inline server/test/endpoint_admin_personas_roles.test.js:13]: @k1ch  just a style question, is there a reason to use a template string here as opposed to just the function `getServerUrl()` ?

**Review by mikelax — APPROVED** [review]
@k1ch  looks good. Should we add optional filter by client as a separate change if/when needed? 

Also, as a separate task we should probably add tests to ensure functional differences are working as expected between full "usher" admins and "client" admins.

File: server/test/endpoint_admin_personas_roles.test.js (hunk 1 above)
Comment (k1ch) [reply server/test/endpoint_admin_personas_roles.test.js]: Thank you for pointing that out. I fixed it in the other files too.

**Comment by k1ch** [comment]
> @k1ch looks good. Should we add optional filter by client as a separate change if/when needed?
> 
> Also, as a separate task we should probably add tests to ensure functional differences are working as expected between full "usher" admins and "client" admins.

@mikelax we can add the filter functionality as an additional feature in a separate task.

Before considering adding more tests, I suggest we determine whether `client-admin` should have access to the `/personas/*` at all?

**Review by mikelax — APPROVED** [review]

**Pull request merged**